code/modules/events/rogue_drones.dm

```diff
@@ -23,20 +23,21 @@
 	var/msg
 	var/rng = rand(1,5)
 	//VOREStation Edit Start TFF 16/12/19 - Sif -> Virgo 3b
+	//YW EDIT 1/13/2020 - Virgo3b -> Cryogaia
 	switch(rng)
 		if(1)
-			msg = "A combat drone wing operating in close orbit above Virgo 3b has failed to return from a anti-piracy sweep.  If any are sighted, \
+			msg = "A Prototype combat drone wing operating in close orbit above Borealis 1 has failed to return from a anti-piracy sweep.  If any have survived re-entry. \
```

Comment:
```suggestion
			msg = "A prototype combat drone wing operating in close orbit above Borealis 1 has failed to return from a anti-piracy sweep.  If any have survived re-entry. \
```

Reply:
I am a derp. Will change that when I get home. Thanks.